_data/experiences/ai-yamamoto-album.yaml

```diff
@@ -0,0 +1,60 @@
+key: ai-yamamoto-album
+type: tour
+button_label: Ai Yamamoto's Album
+title: Ai Yamamoto’s album, 1902-1925
+subtitle:
+date: 1902-1925
+creator: Ai Yamamoto
+summary:
+short_summary:
+
+more_info:
+  - type: Online catalog
+    text: Ai Yamamoto’s Album
+    url: Ai Yamamoto’s Album
```

Comment:
@camillevilla I think this should be `https://searchworks.stanford.edu/view/vg318sh2718` instead of `Ai Yamamoto’s Album`.